bin/load_experiment_web_cli.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
```

Comment:
Could we have a comment as to what this script is doing? Is it ultimately just creating a record in the experiments table?

Reply:
Plus creating the expDesign files, which is what produces trouble with memory on tomcat. I have added to the env file for scxa prod all the needed variables for this to work.